components/input/RichTextEditor.js

```diff
@@ -1,5 +1,10 @@
 import React from 'react';
 import ReactQuill from 'react-quill';
+import Quill from 'quill';
+
+const Block = Quill.import('blots/block');
```

Comment:
Yeah, I think for now that's OK and I'm not seeing a case where `p` is useful for us.

Reply:
Would be nice to find a better way to set defaults for this component. Now this sets div tag as a default element for every instance of RichTextEditor. Might be useful to allow configuring individually.